cluster_cleanup.sh

```diff
@@ -22,3 +22,16 @@ fi
 if [ "${EPHEMERAL_CLUSTER}" == "minikube" ]; then
   sudo su -l -c "minikube delete" "${USER}"
 fi
+
+pushd "${CAPM3PATH}"
+  git reset --hard
```

Comment:
I think for dev they might need to keep some changes in the local repo that will be removed with reset I would recommend resetting only the config files

Reply:
I see. But this is only for the repos under the go path, and it is only run when you do `make clean`. Is it really necessary to keep changes in there?

Reply:
It is hard to say `necessary` but if the repos under `go/src` are always only copies of the repos that might have some uncommitted changes (tested_repos) that will not be a problem. I have not seen the case of changing multiple repos in the same time in practice but theoretically it is the case where developer might want to keep his uncommitted changes in some of those repos